packages/lesswrong/components/sunshineDashboard/SunshineNewUsersList.tsx

```diff
@@ -18,6 +18,9 @@ const SunshineNewUsersList = ({ terms, allowContentPreview }: {
     itemsPerPage: 60
   });
   const { SunshineListCount, SunshineListTitle, SunshineNewUsersItem, LoadMore } = Components
+
+  const showLoading = totalCount && count && totalCount > count
```

Comment:
`useMulti` returns a `showLoadMore` field, which you should use rather than computing this yourself.

Reply:
Ah nice.